serve.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python2
```

Comment:
Any particular reason we're still using python2? It's already EOL...

Reply:
I've taken this from oc cli-artifacts image as is, I plan to iterate on hosting mechanics in coming releases.

Reply:
Also, IIRC python2 is still supported on RHEL.